page-objects/pages/HMWRKHomeworkPage.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
 
 
     studentEditsTextHomeworkAndSubmits: async function () {
-        await this.switchToSubmissionTab();
+        await this.clickOnSubmissionTab();
         await this.submitSolutionForTheHometask();
     },
     // teacher helpers
```

Comment:
again . Do we need userRole Here? hasTheStudentSubmittedTheTask -> Shouldn't it look like this?  isTaskSubmitted

Reply:
the same here as above

Reply:
will be refactored